meson.build

```
@@ -22,6 +22,7 @@ cairo_dependency = dependency('cairo', version: '>=1.14')
 goocanvas_dependency = dependency('goocanvas-3.0')
 libarchive_dependency = dependency('libarchive')
 json_glib_dependency = dependency('json-glib-1.0')
+dep_libportal = dependency('libportal', required: true)
```

```suggestion
libportal_dependency = dependency('libportal')
```
the dependencies are required by default